doc/chapters/integration-guidelines/10-systemd-service.rst

```diff
@@ -0,0 +1,19 @@
+systemd.service
+===============
+
+As it is mentioned in :ref:`Getting Started <getting-started>` section the user API is implemented by
+the SoftwareContainer Agent as a D-bus service. The API is used to start, configure, and control
```

Comment:
I just reviewed @tobsan's PR and in that part of the User Docs we use "SoftwareContainerAgent", so I think we should do the same here.